code/game/objects/items/weapons/stock_parts.dm

```diff
@@ -139,6 +139,93 @@
 	rating = 3
 	matter = list(MATERIAL_PLASTIC = 3, MATERIAL_GLASS = 1)
 
+
+//one star stock parts (rating 4)
```

Comment:
This commit shouldn't be here.

Reply:
I don’t understand how it works, I made a commit from another branch.